packages/framework/src/Framework/Views/Components/RelatedPublicationsComponent.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyde\Framework\Views\Components;
```

Comment:
This class should be in the Publications package so the core package functions without it, but I can fix that if you want.

Reply:
Please fix that. Thank you.